realmd.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: realmd
+  version: 0.17.1
+  epoch: 0
+  description: DBus service for configuring kerberos and other online identities.
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - gettext-dev
+      - krb5-dev
+      - openldap-dev
+      - pkgconf-dev
+      - polkit-dev
+      - systemd-dev
+      - usrmerge-tool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: e80a3ca95672e91772b70d0d8b3fccc52cd0c868
+      repository: https://gitlab.freedesktop.org/realmd/realmd.git
+      tag: ${{package.version}}
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --disable-silent-rules \
+        --with-distro=none \
+        --with-systemd-unit-dir=${{targets.destdir}}/usr/lib/systemd/system \
+        --disable-doc
+
+  - runs: mkdir -p service && touch service/realmd-none.conf
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - runs: |
+      rm ${{targets.destdir}}/usr/lib/realmd/realmd-distro.conf
+      usrmerge-tool -usr-sbin ${{targets.destdir}}
+
```

Comment:
we should consider a subpackage `${{package.name}}-lang` leveraging `split/locales` pipeline.